BE/src/main/java/team07/airbnb/common/auth/jwt/JwtUtil.java

```diff
@@ -46,7 +45,17 @@ public String generateToken(JwtUserDetails userDetails) throws JsonProcessingExc
                 .compact();
     }
 
-    public TokenUserInfo getUserEntity(String token) throws JsonProcessingException {
+    public boolean validateToken(String token) {
+        try {
+            Jwts.parser().verifyWith(key).build().parseSignedClaims(token);
```

Comment:
```suggestion
            Jwts.parser()
                .verifyWith(key)
                .build()
                .parseSignedClaims(token);
```
- 가독성을 위해 점(.) 은 한줄에 하나만 찍는건 어떨까요? (제안드려봅니다) 🤗